server/poetry.lock

```diff
@@ -730,13 +730,13 @@ files = [
 
 [[package]]
 name = "django"
-version = "4.2.16"
+version = "4.2.17"
```

Comment:
In general it is best to keep package updates to separate PRs, unless relevant to the PR

Reply:
I'm not sure how this happened. It might have come from switching the branch to be merged into from `main` to the Compress branch.
I'll get that changed.